FALSEHOODS.md

```diff
@@ -118,3 +118,18 @@ surprising how many false assumptions programmers continue to make about them.
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  A given phone number has only one associated phone line. (Corollary: if a
```

Comment:
I'm not sure how I understand how this assumption could cause programmers to make mistakes: what could they be doing wrong if they wrongly assume this?